code/game/turfs/simulated/wall_natural.dm

```diff
@@ -0,0 +1,161 @@
+var/list/strata_by_z = list()
+var/list/strata_material_by_z = list()
```

Comment:
Confusing name: it's not by z, it's by some key system you have to know. Though after reading it, if it's the case that every z level has a unique strata, this could be by z instead, probably?

Reply:
What is confusing about it? The key is the strata and the z-level. A z-level can have multiple stratas via turf mapping, the `strata_by_z` list is the default strata.

Reply:
Renamed them to be clearer.